README.md

```diff
@@ -0,0 +1,95 @@
+We are developing an international ERP and a platform that supports business development and scalable growth for our partners and their customers.
+
+To more information visit [etendo.software](https://etendo.software)
+
+### Requirements
+In this section you can read the [System Requirements](https://docs.etendo.software/en/technical-documentation/etendo-environment/requirements-and-tools/requirements).
+
+### Etendo Instalation
+Etendo includes the Classic ERP functionalities and Etendo RX, a reactive platform to execute microservices able to interact with the database and perform asynchronous actions.
+
+1. Clone the repository and move to develop branch
+```
+git clone  -b develop git@github.com:etendosoftware/etendo.git
+```
+2. To compile and deploy an Etendo instance you have to setup the configuration variables, to do that you have to create a copy of `gradle.properties.template` file in root and src-rx folders.
+```bash
+cp gradle.properties.template gradle.properties
+cp src-rx/gradle.properties.template src-rx/gradle.properties
+```
+2. You can edit `gradle.properties` files updating the variables or use the default values
+
+>The Nexus user and Password is required
+
+| Variable                     | Description                                                               | Default value |
+|------------------------------|---------------------------------------------------------------------------|---------------|
+| nexusUser <br> nexusPassword | Nexus repository credentials, for access to commercial modules            |               |
+| context.name                 | Environment name                                                          | etendo        |
+| bbdd.sid                     | Database name                                                             | etendo        |
+| bbdd.port                    | Database port                                                             | 5432          | 
+| bbdd.systemUser              | Database system user                                                      | postgres      |
+| bbdd.systemPassword          | Database system password                                                  | syspass       |
+| bbdd.user                    | Database user                                                             | tad           |
+| bbdd.password                | Database password                                                         | tad           |
+
+> Run the gradle tasks with the `--info` or `--debug` flag to log more information.
+
+Change the repositoryURL to etendo-snapshot-jars
+```
+...
+org.gradle.jvmargs=-Xmx2g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+repositoryUrl=https://repo.futit.cloud/repository/etendo-snapshot-jars
+repositoryUser=
+repositoryPassword=
+...
+```
+
+4. Run setup tasks to create the configurations files
+```
+./gradlew setup
+```
+4. Change the port to 5432 in the `src-rx/rxconfig/das.yaml` file
```

Comment:
Is the port always 5432 or does it have to match the one put in the gradle.properties file?

Reply:
It has to match the one put in the gradle.properties file, solved